atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockIntegrationTest.java

```diff
@@ -119,15 +123,35 @@ public void shouldWrapCheckedExceptionsInARuntimeException() {
     }
 
     @Test
-    public void testLocksTimeout() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
This was removed since we no longer timeout when waiting on the lock. We now have 2 possible states - when the heartbeat is alive and when its dead. When it's alive, we keep retrying (with backoff) until the lock is released. When its no longer detected as alive, we attempt to grab the lock assuming that the lock holder is dead.

We may still want to keep timeouts around though for cases where the action might never terminate. @tpetracca ?